libs/block-content/website/src/lib/bildwurf-ad-block.tsx

```diff
@@ -0,0 +1,92 @@
+import {useEffect, useState, useRef, useCallback, useMemo} from 'react'
+import {BildwurfAdBlock as BildwurfAdBlockType, Block} from '@wepublish/website/api'
+
+import styled from '@emotion/styled'
+import {BuilderBildwurfAdBlockProps} from '@wepublish/website/builder'
+
+function useScript(src: string, checkIfLoaded: () => boolean, crossOrigin = false) {
+  const scriptRef = useRef<HTMLScriptElement | null>(null)
+
+  const [isLoading, setLoading] = useState(false)
+  const [isLoaded, setLoaded] = useState(() => checkIfLoaded())
+
+  useEffect(() => {
```

Comment:
Can we use `<Script>` instead? (It's in the website builder, probably won't work on storybook but should in the next app as the storybook mock is very basic)